libs/datasets/data_source.py

```diff
@@ -127,6 +127,8 @@ def make_dataset(cls) -> timeseries.MultiRegionDataset:
                 .last()
                 .reset_index()
                 .drop(columns=[CommonFields.DATE])
+                .copy()
             )
+            timeseries._tag_df_add_all_bucket_in_place(source_tag_df)
```

Comment:
since we're accessing this outside of the class, would prefer if it was just `tag_df_...`